### DISCUSSION TITLE
Deepin Browser Source Code

### DISCUSSION THREAD

**Discussion by suleymancetin8** [linuxdeepin/developer-center#10997] [General | 常规讨论]
Source code of Deepin Browser isn't provided. The repository is empty. Why? 
Browser is one of the most critical part of an operating system when it comes to privacy. Since Deepin claims that Deepin OS is open source, the source code of Deepin Browser MUST be open for community!

**Comment by BLumia** [comment]
> Since Deepin claims that Deepin OS is open source

Well, the major part of the OS (the desktop environment *DDE*, most DTK-based deepin applications, most of the system packages) are open-sourced, but still not all parts of the OS are open-sourced. There *are* close-sourced components and I think the browser project is one of them.

It's yet another browser based on chromium, so IMHO if the browser team don't plan to release the source, you can, well, simply switch to chromium or firefox.

**Comment by suleymancetin8** [comment]
why switching? deepin is a well designed linux operating system and it can be fully open source. I hope you will share the source code of the whole operating system and show the world the power of open source! let's prove the world that deepin os is pure goodness!

**Reply by LFRon** [reply]
> why switching? deepin is a well designed linux operating system and it can be fully open source. I hope you will share the source code of the whole operating system and show the world the power of open source! let's prove the world that deepin os is pure goodness!

Actually as a deepin v23 user,the deepin browser just the same as Firefox and Thorium,which doesn't include proprietary function.You can uninstall it right now and install firefox in the app store or Flathub which will not cause any effection.

**Reply by suleymancetin8** [reply]
> > why switching? deepin is a well designed linux operating system and it can be fully open source. I hope you will share the source code of the whole operating system and show the world the power of open source! let's prove the world that deepin os is pure goodness!
> 
> Actually as a deepin v23 user,the deepin browser just the same as Firefox and Thorium,which doesn't include proprietary function.You can uninstall it right now and install firefox in the app store or Flathub which will not cause any effection.

You really don't understand what the spirit of open source is.
